CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 - Username formatting now correctly only prints FullUsername field ([#126](https://github.com/cyberark/conjur-authn-k8s-client/issues/126))
+- Timeout for token retrievals is now correctly reset in consequent failures
+  ([cyberark/conjur-authn-k8s-client#158](https://github.com/cyberark/conjur-authn-k8s-client/issues/158))
+
+### Changed
+- Wait slightly for the client certificate file to exist after login before
+  raising an error.
```

Comment:
What is the user impact of this change? How does it affect the UX?

Reply:
the client will fail less frequently as we now wait for up to 10 seconds to read the cert instead of immediately reading it. We are also logging our efforts so that's another customer-facing change. would you change the entry according to this?